tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2021",
+    "target": "es2022",
```

Comment:
I believe we chose ES2021 to support Node.js 16

Reply:
Es 2021 doesn't support private methods that we already use. This still passes with node 16 with 2022.